onlineShopWeb/Models/HomeViewModels.cs

```diff
@@ -10,4 +10,14 @@ public class HomeViewModel
         public List<Song> SongsList { get; set; }
         
     }
+
+    public class HomeViewModelBook
```

Comment:
Why do you need this class if it only has a Book inside? Why not use Book directly